geth/api/backend.go

```diff
@@ -34,12 +41,14 @@ func NewStatusBackend() *StatusBackend {
 	accountManager := account.NewManager(nodeManager)
 	txQueueManager := txqueue.NewManager(nodeManager, accountManager)
 	jailManager := jail.New(nodeManager)
+	notificationManager := notification.New(notification.NewFCMProvider(fcm.NewFcmClient(fcmServerKey)))
```

Comment:
Could it work like this:
```go
notification.NewFCMProvider(fcmServerKey)
```
so that we encapsulate client creation in `NewFCMProvider`?